File: exercicios/cpp/Instruções Sequenciais/Q16.cpp
/*
------------------------------------------------
16. Crie um algoritmo/fluxograma/programa que 
solicite ao usuário o raio da base e a altura 
de uma caixa d'água cilíndrica, calcule, e 
imprima na tela o seu volume em litros. 
Usar a biblioteca cmath.
 */
#include <iostream>
#include <cmath>
#include <clocale>

using namespace std;
int main () {
	setlocale(LC_ALL,"Portuguese");
	double area, raio, altura;
	cout<<"Informe o raio (m):";
	cin>>raio;
	cout<<"Informe a altura (m):";
	cin>>altura;    
    area = M_PI * pow(raio, 2) * altura;
    cout<<"Volume:"<<area<<" m³"<<endl;
	return 0;
}

File: exercicios/python/Instruções Sequenciais/Q16.py
# Exercício 16 — Volume da caixa d'água (cilíndrica) em litros
import math

def main():
    raio = float(input("Raio da base (m): "))
    altura = float(input("Altura da caixa (m): "))
    volume_m3 = math.pi * raio**2 * altura
    volume_litros = volume_m3 * 1000
    print(f"Volume: {volume_litros:.2f} litros")

if __name__ == "__main__":
    main()